src/Neo/Network/P2P/Capabilities/NodeCapabilityType.cs

```diff
@@ -25,12 +23,6 @@ public enum NodeCapabilityType : byte
         /// </summary>
         TcpServer = 0x01,
 
-        /// <summary>
-        /// Indicates that the node is listening on a WebSocket port.
-        /// </summary>
-        [Obsolete]
-        WsServer = 0x02,
```

Comment:
I don't think it's correct to remove it like this. Technically it existed in the P2P protocol, so at least the number should be kept as reserved.

Reply:
ObsoleteWsServer

Reply:
we can always repurpose it in the future. Reserving it says to me that it still `WsServer` or active.

Reply:
I do not agree, either we should implement it or w should remove it, having somthing in the core that is not implemented is not a good practice, might exist in the past by mistake, but should be fixed, there is no hardfork for this, should not bother.